src/pages/patientView/clinicalInformation/getClinicalInformationData.ts

```diff
@@ -0,0 +1,106 @@
+import queryString from 'query-string';
+import { chain } from 'underscore';
+import CBioPortalAPI from "shared/api/CBioPortalAPI";
+import { getTreeNodesFromClinicalData } from './PDXTree';
+import {ClinicalDataSummary} from "../../../shared/api/CBioPortalAPI";
+import {SampleSummary} from "../../../shared/api/CBioPortalAPI";
+//import sampleQuery from 'shared/api/mock/Samples_query_patient_P04.json';
+
+
+type TODO = any;
+
+/*
+ * Transform clinical data from API to clinical data shape as it will be stored
+ * in the store
+ */
+function transformClinicalInformationToStoreShape(patientId: string, studyId: string, clinicalDataPatient: Array<ClinicalDataSummary>, clinicalDataSample: Array<ClinicalDataSummary>) {
+    const patient = {
```

Comment:
We need a patient type. But patient seems likely to be a frequently used entity in our system.  Where should this type exist?  Should it have different properties depending on where it's used?